components/editor/TimesheetEntryEditor.tsx

```diff
@@ -149,6 +163,10 @@ const DayEditor = ({
                             deleteTimesheetEntry={del}
                             dateStr={dateStr}
                             timeCategories={timeCategories}
+                            tasks={taskByProject(
+                                tasks,
+                                entry.timesheet.project.id || 0
```

Comment:
I think it would be slightly better semantically, if this was conditionally rendered and `project.id` would not default to zero:

```typescript
                       entry.timesheet.project.id && <TimesheetEntryRow
                            entry={entry}
                            deleteTimesheetEntry={del}
                            dateStr={dateStr}
                            timeCategories={timeCategories}
                            tasks={taskByProject(
                                tasks,
                                entry.timesheet.project.id 
                            )}
                        />
```